packages/medusa/src/loaders/plugins.ts

```diff
@@ -324,6 +361,15 @@ export async function registerServices(pluginDetails: PluginDetails, container:
           ),
           [`fileService`]: aliasTo(name),
         })
+      } else if (loaded.prototype instanceof AbstractFileService) {
```

Comment:
suggestion: Maybe the util `isFileService` could take care to check if it is instance of AbstractFileService or FileService and use that util here